pwrsupply/pstypes-names.txt

```diff
@@ -87,7 +87,8 @@ si-vping                     bipolar          corrector-vertical
 bo-dclink-fac-2s             none            none
 bo-dclink-fac-2p4s           none            none
 as-dclink-fbp                none            none
-as-dclink-regatron           none            none
+as-dclink-regatron-master    none            none
```

Comment:
terms master and slave are in disuse. [This link](https://www.theserverside.com/opinion/Master-slave-terminology-alternatives-you-can-use-right-now) has a list of possible substitutes.

Reply:
The use here follow Regatron/ELPS naming